README.md

```diff
@@ -41,4 +83,9 @@ new doctest-able modules are added to the CI.
 
 This file can be run from the shell using `python tests/run_doctest.py`.
 
-[sampleproject]: https://github.com/Chris-Johnston/Easier68k-SampleProject
\ No newline at end of file
+[sampleproject]: https://github.com/Chris-Johnston/Easier68k-SampleProject
```

Comment:
Very minor, but I think it's best to keep these types of links at the very end of the document.